src/pybind/mgr/influx/module.py

```diff
@@ -178,12 +178,12 @@ def get_df_stats(self, now):
         pool_info = {}
 
         df_types = [
-            'bytes_used',
+            'stored_used',
```

Comment:
```
2018-11-28T01:31:01.553 INFO:tasks.ceph.mgr.x.smithi131.stderr:2018-11-28 01:31:01.525 7f3546fb3700 -1 Remote method threw exception: Traceback (most recent call last):
2018-11-28T01:31:01.553 INFO:tasks.ceph.mgr.x.smithi131.stderr:  File "/usr/lib64/ceph/mgr/influx/module.py", line 427, in self_test
2018-11-28T01:31:01.553 INFO:tasks.ceph.mgr.x.smithi131.stderr:    df_stats, pools = self.get_df_stats(now)
2018-11-28T01:31:01.553 INFO:tasks.ceph.mgr.x.smithi131.stderr:  File "/usr/lib64/ceph/mgr/influx/module.py", line 207, in get_df_stats
2018-11-28T01:31:01.554 INFO:tasks.ceph.mgr.x.smithi131.stderr:    "value": pool['stats'][df_type],
2018-11-28T01:31:01.554 INFO:tasks.ceph.mgr.x.smithi131.stderr:KeyError: ('stored_used',)
```

http://pulpito.ceph.com/kchai-2018-11-28_00:59:04-rados-wip-kefu2-testing-2018-11-27-2345-distro-basic-smithi/3286665/

Reply:
i think it should be `stored`.

Reply:
@tchaikov - indeed. Thanks, Kefu!